lib/host.py

```diff
@@ -53,7 +54,8 @@ def start(self):
           idle = random.randint(5, 10)
           print("Lock not available, waiting %s seconds"%idle)
           time.sleep(idle)
-    ret = os.system(proxy + "vagrant up " + self.hostid + " --provider="+self.provider)
+    command = proxy + "vagrant up " + self.hostid + " --provider="+self.provider
+    ret = subprocess.call(command.split(" "))
```

Comment:
os.system keeps stdout and stderr to terminal, whereas subprocess call removes it

Reply:
There is no need for this whe you use the call method, see https://docs.python.org/fr/3/library/subprocess.html#subprocess.call
It does not removes stdout and stderr by default